### ISSUE TITLE
Control multiple players at once

### ISSUE THREAD

**Issue by kuviman** [kuviman/linksider#39]
Color coded? control all of same color, switch colors

**Comment by SHADOW-CRUSHERZ** [comment]
good idea.... actuzlly great idea, but i think would be hard with level design

**Comment by DaivyIsHere** [comment]
colored goal that only allows the player with same color to reach
